account_financial_report/wizard/journal_ledger_wizard_view.xml

```diff
@@ -7,38 +7,56 @@
-                </group>
-                <separator string="Options" />
-                <group name="options">
-                    <group>
-                        <field
-                            name="move_target"
-                            widget="radio"
-                            options="{'horizontal': true}"
-                        />
-                        <field name="sort_option" />
-                        <field name="group_option" />
-                        <field name="foreign_currency" />
-                        <field name="with_account_name" />
-                        <field name="with_auto_sequence" />
-                    </group>
-                    <group />
-                </group>
-                <separator string="Journals" />
-                <group>
-                    <field name="journal_ids" widget="many2many_tags" />
-                </group>
+               <sheet>
+                    <notebook>
+                       <page string="Filters">
+                            <group>
+                                <field
+                                    name="company_id"
+                                    groups="base.group_multi_company"
+                                />
+                            </group>
+                            <separator string="Periods" />
+                            <group>
+                                <group>
+                                    <field name="date_range_id" />
+                                    <field name="date_from" />
+                                    <field name="date_to" />
+                                </group>
+                                <group />
+                            </group>
+                            <separator string="Options" />
+                            <group name="options">
+                                <group>
+                                    <field
+                                        name="move_target"
+                                        widget="radio"
+                                        options="{'horizontal': true}"
+                                    />
+                                    <field name="sort_option" />
+                                    <field name="group_option" />
+                                    <field name="foreign_currency" />
+                                    <field name="with_account_name" />
+                                    <field name="with_auto_sequence" />
+                                </group>
+                                <group />
+                            </group>
+                            <separator string="Journals" />
+                            <group>
+                                <field name="journal_ids" widget="many2many_tags" />
+                            </group>
+                        </page>
+                        <page string="Columns">
+                            <group name="columns">
```

Comment:
```suggestion
                            <group name="columns">
```
Remove the group for better visibility (full width). This applies to all wizards (except general ledger).

Reply:
Removed